pdns/dnsdistdist/dnsdist-tcp-upstream.hh

```diff
@@ -170,6 +172,7 @@ static void handleTimeout(std::shared_ptr<IncomingTCPConnectionState>& state, bo
   size_t d_proxyProtocolNeed{0};
   size_t d_queriesCount{0};
   size_t d_currentQueriesCount{0};
+  std::thread::id d_mainThreadID;
```

Comment:
I find this name a bit confusing, as "main thread" is commonly used for the first thread of a process. Suggestions: `d_creatorThreadID`?